custom/icds_reports/reports/poshan_progress_dashboard_data.py

```diff
@@ -175,7 +186,7 @@ def get_poshan_progress_dashboard_data(domain, year, month, quarter, data_format
 
     if step == 'aggregated':
         data = queryset.aggregate(
-            count=Count('state_name'),
+            count=Count('state_name', distinct=True),
```

Comment:
could you please tell why this is hardcoded to state_name?

Reply:
oh because state_name are needed, when it's national level

Reply:
and when its state level, dont we want district names?